tests/Nhs.Appointments.Core.UnitTests/UserServiceTests.cs

```diff
@@ -89,14 +89,14 @@ public async void IncludesUserIdInEvent()
         }
 
         [Fact]
-        public async void ReturnsFailureWhenRoleNotFound()
+        public async Task ReturnsFailureWhenRoleNotFound()
```

Comment:
What's the advantage of this?

Reply:
"Support for 'async void' unit tests is being removed from xUnit. net v3" is the warning this generates - I haven't really looked into exactly why but I was lead to believe that `async Task` is preferable to `async void` anyway.

From a very quick cursory Google it sounds like there are differences in how exceptions are handled in each case, and the test framework might struggle to observe when an `async void` method has completed. I can look into it further if you like